src/components/ui/DropdownMenu/dropdownMenu.scss

```diff
@@ -1,63 +1,71 @@
-.Layer__dropdown-menu__popover {
-    outline: none;
-    &[data-entering] {
-        animation: popover-slide 120ms;
-    }
+.Layer__UI__DropdownMenu__Popover {
+  &[data-entering] {
+    animation: popover-slide 120ms;
+  }
 
-    &:focus-visible {
-        outline: none;
-    }
+  &[data-exiting] {
+    animation: popover-slide reverse 120ms;
+  }
 }
 
 @keyframes popover-slide {
-    from {
-      transform: translateY(-20px);
-      opacity: 0;
-    }
-  
-    to {
-      transform: translateY(0);
-      opacity: 1;
-    }
+  from {
+    transform: translateY(-20px);
+    opacity: 0;
   }
 
-.Layer__dropdown-menu__menu {
-    padding-top: var(--spacing-xs);
-    padding-bottom: var(--spacing-xs);
-    background: var(--color-base-0);
-    border-radius: var(--border-radius-xs);
-    box-shadow: 0px 0px 0px 1px rgba(25, 25, 25, 0.06), 0px 4px 12px 0px rgba(17, 17, 17, 0.01), 0px 2px 4px 0px rgba(25, 25, 25, 0.06);
+  to {
+    transform: translateY(0);
+    opacity: 1;
+  }
 }
 
-.Layer__dropdown-menu__menu-item {
-    display: flex;
-    align-items: center;
-    gap: var(--spacing-xs);
-    padding: var(--spacing-xs);
-    border-radius: var(--border-radius-3xs);
-    box-shadow: 0px 0px 0px 1px var(--base-transparent-4);
-    margin: var(--spacing-3xs) var(--spacing-xs);
-    cursor: pointer;
-    transition: background-color 0.1s ease-in-out;
-
-    &:hover,
-    &:focus-visible {
-        background-color: var(--color-base-50);
-    }
-}
+.Layer__UI__DropdownMenu__Dialog {
+  display: grid;
+  gap: var(--spacing-xs);
+  overflow: hidden;
+  padding: var(--spacing-sm);
+  border-radius: var(--border-radius-xs);
+  border: 1px solid var(--border-color);
+  background: var(--color-base-0);
 
-.Layer__dropdown-menu__menu-list {
-    padding-bottom: var(--spacing-3xs);
+  &[data-variant='compact'] {
+    padding: 0;
+  }
 }
 
-.Layer__dropdown-menu__separator {
-    height: 1px;
-    width: 100%;
-    margin: var(--spacing-xs) 0;
-    background-color: var(--color-base-200);
+.Layer__UI__DropdownMenu__MenuItem {
```

Comment:
[This is from our design system](https://www.figma.com/design/2xTjR7w56ebynNx8iGoQCu/Layer---design-system?node-id=9802-38647&t=lqwhJQc8JvNrbaV6-4) - as pictured its for a filter actions menu, but I believe it's intended to be used for general hamburger menu style dropdowns like this